src/xcode/ENA/ENA/Source/Services/DCCValidationService/Model/DCCValidationError.swift

```diff
@@ -0,0 +1,30 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+enum DCCValidationError: Error {
+	
+	// MARK: - Internal
+	
+	case TECHNICAL_VALIDATION_FAILED (DCCValidationProgress)
+	case ACCEPTANCE_RULE_CLIENT_ERROR (DCCValidationProgress)
+	case ACCEPTANCE_RULE_JSON_ARCHIVE_FILE_MISSING (DCCValidationProgress)
+	case ACCEPTANCE_RULE_JSON_ARCHIVE_SIGNATURE_INVALID (DCCValidationProgress)
+	case ACCEPTANCE_RULE_JSON_EXTRACTION_FAILED (DCCValidationProgress)
+	case ACCEPTANCE_RULE_SERVER_ERROR (DCCValidationProgress)
+	case INVALIDATION_RULE_CLIENT_ERROR (DCCValidationProgress)
+	case INVALIDATION_RULE_JSON_ARCHIVE_FILE_MISSING (DCCValidationProgress)
+	case INVALIDATION_RULE_JSON_ARCHIVE_SIGNATURE_INVALID (DCCValidationProgress)
+	case INVALIDATION_RULE_JSON_EXTRACTION_FAILED (DCCValidationProgress)
+	case INVALIDATION_RULE_SERVER_ERROR (DCCValidationProgress)
+	case ONBOARDED_COUNTRIES_CLIENT_ERROR (DCCValidationProgress)
+	case ONBOARDED_COUNTRIES_JSON_ARCHIVE_FILE_MISSING (DCCValidationProgress)
+	case ONBOARDED_COUNTRIES_JSON_ARCHIVE_SIGNATURE_INVALID (DCCValidationProgress)
+	case ONBOARDED_COUNTRIES_JSON_EXTRACTION_FAILED (DCCValidationProgress)
+	case ONBOARDED_COUNTRIES_SERVER_ERROR (DCCValidationProgress)
```

Comment:
Maybe we can extract these into a `DCCOnboardedCountriesError` and double up the `NO_NETWORK` error for that

Reply:
I will have a look.

Reply:
Okay that makes sense to extract all the onboarded_countries_errors out from here to separate it for the 2 http calls.